### ISSUE TITLE
ConvergenceWarning

### ISSUE THREAD

**Issue by bberzh** [BetterBoyTph/SVM-CIFAR10#1]
When I run  SVM_CIFAR10.py, occur following error:
max_accuracy=0.371000,best_learning_rate=1.000000e-07,best_regularization_strength=7.000000e+04
The test accuracy with self-realized svm is:0.362000

Program time of self-realized svm is:16285.736093000001s
.../lib/python3.6/site-packages/sklearn/svm/_base.py:986: ConvergenceWarning: Liblinear failed to converge, increase the number of iterations.
  "the number of iterations.", ConvergenceWarning)
The test accuracy with svm.LinearSVC is:0.172000
Program time of svm.LinearSVC is:972.2910360000005s

**Comment by bberzh** [comment]
And I don‘t know where to change.
